src/client/mixins/basic-interaction.ts

```diff
@@ -4,6 +4,19 @@ import { Promise as Bluebird } from 'bluebird';
 export class BasicInteractionAware {
   public client: Page;
 
+  public async scrollThrough(depth: number) {
+    try {
+      await this.client.evaluate(
+        (percent) => {
+          const floatValue = percent / 100;
+          window.scrollBy(0, document.body.scrollHeight * floatValue);
+        },
+        depth);
+    } catch (e) {
+      throw Error(`Unable to scroll through ${depth} percent depth`);
```

Comment:
It could be useful to get the string value of the error caught here--if it ever pops up, I'd want to know what it was.